app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -33,6 +35,13 @@ export const CollectionPage = (): React.ReactElement => {
     updateCollection();
   }, [updateCollection]);
 
+  React.useEffect((): void => {
+    setCollectionStatistics(null);
+    notdClient.getCollectionStatistics(address).then((getCollectionStatistics: CollectionStatistics): void => {
```

Comment:
you dont deal with errors here - please copy paste from nftcard and edit, dont write stuff from scratch. the variable name should be retrieved, its not related to the function.